docs/README.md

```diff
@@ -80,13 +80,13 @@ Next generation available: <End-date>
 
 To make a force restart, add the `force` argument to the query, as in: `generateSitemap(force: true)`. But, be aware that this will cancel the previous process.
```

Comment:
```suggestion
   To make a force restart, add the `force` argument to the query, as in: `generateSitemap(force: true)`. But, be aware that this will cancel the previous process.
```